### ISSUE TITLE
Not able to find SKIP directive

### ISSUE THREAD

**Issue by kinow** [tupilabs/tap4j#37]
In the following TAP stream

```
#cat /var/lib/jenkins/jobs/gh-mellanox-v1.8-PR/builds/137/tap-master-files/cov_stat.tap
not ok - coverity detected 1411 failures in all # SKIP
ok - coverity found no issues for oshmem
ok - coverity found no issues for yalla
ok - coverity found no issues for mxm
ok - coverity found no issues for fca
ok - coverity found no issues for hcoll
```
The parser is not able to find the SKIP directive.

**Comment by kinow** [comment]
Looks like the REGEX_TEST_RESULT, in the Constants.java, needs to replace `([^#]+)` by `([^#]*)`

**Comment by kinow** [comment]
Fixed in 4.1.1
